### ISSUE TITLE
Pretraining on ImageNet

### ISSUE THREAD

**Issue by tarun005** [thuml/Universal-Domain-Adaptation#9]
You had mentioned that the backbone network is ResNet-50 pretrained on Imagenet. 

https://github.com/thuml/Universal-Domain-Adaptation/blob/5d7caa95af7e3675305c542253c4e372801897d2/net.py#L37

But in many experiments in the paper, the labels in the unsupervised target overlap with that of the supervised source labels in the ImageNet. Is it justified that you pretrain on supervised ImageNet labels?

**Comment by youkaichao** [comment]
That's a nice point. We use pre-trained models because they are so popular and are used by default. Maybe you can investigate how pre-trained models affect the performance :)
